keylime/tpm2.py

```diff
@@ -133,7 +136,7 @@ def __get_tpm_algorithms(self):
 
         if tools_version == "3.2":
             retDict = self.__run("tpm2_getcap -c algorithms")
-        elif tools_version == "4.0":
+        elif tools_version == "4.0" or tools_version == "4.2" :
```

Comment:
tools_version in ["4.0", "4.2"] ?

Reply:
done, pushed a new commit with this suggested modification (for all entries)